test/nuts/perfResults/x64-linux-2xIntel-Xeon(R)-Platinum-8171M-CPU--2-60GHz/eda.json

```diff
@@ -0,0 +1,18 @@
+[
```

Comment:
Can/should perfResults be added to the .gitignore?

Reply:
no, we want those checked in so that a PR compares its results to what's on main (to detect significant perf regressions/improvements)

Reply:
Might not be a bad idea to create a README in this folder explaining how these files are generated and what they are used for (or add it to the top level README)